optimizely/client/factory_test.go

```diff
@@ -48,41 +49,23 @@ func TestFactoryClientReturnsDefaultClient(t *testing.T) {
 func TestClientWithSDKKey(t *testing.T) {
 	factory := OptimizelyFactory{SDKKey: "1212"}
 
-	clientOptions := Options{}
-
-	client, err := factory.ClientWithOptions(clientOptions)
+	app, err := factory.Client()
```

Comment:
Not: instead of app it can just be optimizelyClient. I think that is a more accurate name for it.

Reply:
done